### DISCUSSION TITLE
How should localhost be fully setup?

### DISCUSSION THREAD

**Discussion by jhonforde** [edumeet/edumeet#828] [Install/Run/Config]
I downloaded the project into localhost and applied the following codes in order:
1) Copy server / config / config.example.js to server / config / config.js
2) Copy app / public / config / config.example.js to app / public / config / config.js
3) cd app
4) npm install
5) npm run build
6) cd ..
7) cd server
8) npm install

and I could not run the project. How should the project be setup?

**Comment by havfo** [comment]
Regarding your points 1 and 2, please read the [manual installation](https://github.com/edumeet/edumeet#manual-installation) part of the readme.

**Comment by misi** [comment]
@jhonforde So sorry, but please share more info. 
We can't help without a very detailed description of the error you experience.. :(

**Comment by prayaslashkari** [comment]
@misi facing the same issue. Regarding the _"Edit your two config.js with appropriate settings (listening IP/port, logging options, valid TLS certificate, don't forget ip setting in last section in server config: (webRtcTransport), etc)."_

What exactly can i edit to make it work?
